redis/README.md

```diff
@@ -31,3 +37,31 @@ OPSLEVEL_APP_TIER="tier_4" # Tier of your Redis in OpsLevel- see https://wiki.uw
 REDIS_SECRET_NAME="redis" # Name of the secret with your Redis password. See secret created in directory `example`.
 REDIS_REPLICA_COUNT=1 # Amount of read-only replicas 
 ```
+
+## Usage
+In order to manually access the database:
+1. [Install redis-cli](https://redis.io/docs/install/install-redis/) (it comes together with server installation)
+2. port-forward to Redis service running in the cluster:
+    ```bash
+    kubectl port-forward svc/redis-shared-master 6379
+    ```
+3. Connect redis-cli to service:
+    ```bash
+    redis-cli -u redis://localhost:6379 -a "password" 
+    ```
+4. Use Redis console. For example, to get the list of all the keys, run
+    ```redis
+    KEYS *
+    ```
+
+## Metrics
+There are Grafana dashboards prepared for the Redis instances installed via shared kustomize base.
+They can be found using the name [redis overview](https://grafana.prod.aws.uw.systems/goto/8N_RY8OSg?orgId=1).
```

Comment:
What about a dashboard for other clusters (e.g. `dev-aws`)?

Reply:
Should we really list the links for each of environments? I trust in the developers skills and I assume they would be able to find it themselves, example of one environment should be enough

Reply:
> Should we really list the links for each of environments? I trust in the developers skills and I assume they would be able to find it themselves, example of one environment should be enough

yeah, listing them all is a bit too much, could possibly change things up a bit (but :shrug: maybe not more informative):

```suggestion
They can be found using the name "redis overview", e.g. [for `prod-aws`](https://grafana.prod.aws.uw.systems/goto/8N_RY8OSg?orgId=1).
```

Reply:
Neat idea :)